### ISSUE TITLE
Sentence Segmenter

### ISSUE THREAD

**Issue by akash435** [prathimacode-hub/Awesome_Python_Scripts#741]
---
Title: Sentence Segmenter
About: It's an GUI script
Name: Akash Ramanand Rajak
Label: Feature Request
Assignee: ''

---

Define You:

- [x] LGM-SOC'21 Participant 
- [ ] DevIncept Participant
- [ ] Contributor

In this script, will create an application using tkinter GUI,

In this application user will be able to segments the paragraphs into multiple sentence.

This sentence segmenting will be done on the basis of identifying the sentence boundaries.

Basically here user will be first allowed to enter any paragraph, and the script will split the paragraph into sentences by identifying different sentence boundaries.

**Comment by akash435** [comment]
@prathimacode-hub please assign this to me under LGM SOC.

**Comment by prathimacode-hub** [comment]
Issue assigned. @akash435
